frontend/src/queries/schema.ts

```diff
@@ -124,8 +125,9 @@ export interface HogQLQuery extends DataNode {
 export interface EntityNode extends DataNode {
     name?: string
     custom_name?: string
-    math?: BaseMathType | PropertyMathType | CountPerActorMathType | GroupMathType
+    math?: BaseMathType | PropertyMathType | CountPerActorMathType | GroupMathType | HogQLMathType
     math_property?: string
+    math_hogql?: string
     math_group_type_index?: 0 | 1 | 2 | 3 | 4
```

Comment:
In terms of data modeling, this should really be just one `math` property, as it doesn't make sense for e.g `math_hogql` to be set if `math` is not `HogQLMathType` – or vice versa.
Ideally this'd be like:
`math: TrendsMath` and then `TrendsMath` would be `{ type: HogQLMathType, expression: string } | { type: PropertyMathType, property: string } | ...`
Not for this PR though I suppose.